src/app/caesar-portal/profile/profile.component.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpModule } from '@angular/http';
+
+import { ProfileComponent } from './profile.component';
+import { ProfileService } from './profile.service';
+import { User } from '../common/models/user';
+
+
+describe('ProfileComponent', () => {
+    let component: ProfileComponent;
+    let fixture: ComponentFixture<ProfileComponent>;
+
+    beforeEach(async(() => {
+        const profileServiceStub = {
```

Comment:
this stub is passed instead of service, so you should add the method you've used from this service to the stub: ie getCurrentUser